network/policy/policy_windows.go

```diff
@@ -358,3 +358,12 @@ func GetHcnEndpointPolicies(policyType CNIPolicyType, policies []Policy, epInfoD
 
 	return hcnEndPointPolicies, nil
 }
+
+// AddDnsNATPolicy adds DNS NAT policy and returns serialized json
+func AddDnsNATPolicy() (json.RawMessage, error) {
+	outBoundNatPolicy := hcsshim.OutboundNatPolicy{
+		Policy:       hcsshim.Policy{Type: hcsshim.OutboundNat},
+		Destinations: []string{"168.63.129.16"}}
```

Comment:
move dns address to constant

Reply:
this is the only location, and it's a temporary change